src/app/search/search.component.ts

```diff
@@ -410,6 +420,36 @@ export class SearchComponent implements OnInit, OnDestroy {
       .catch((error) => console.log(error));
   }
 
+  updateResultsTableSeparately(toolsQuery: string, workflowsQuery: string) {}
+  /**
+   * Updates the results table when there is no search term
+   * We need to send one request per index
+   * When each index returns its results, join the results into a single array and filter into table
+   *
+   * @param {string} value the elastic search query
+   * @memberof SearchComponent
+   */
+  updateResults(toolsQuery: string, workflowsQuery: string) {
+    const tools = ELASTIC_SEARCH_CLIENT.search({
+      index: 'tools',
+      type: 'entry',
+      body: toolsQuery,
+    }).catch((error) => console.log(error));
```

Comment:
This was like this, but I think here and on line 422 we should be surfacing the error to the user, rather than just silently failing. Everything that can fail will fail at some point. :) Could you please log it as a followup ticket?

Reply:
Ok, logged here [3952](https://github.com/dockstore/dockstore/issues/3952)